test/Java8andUp/src/org/openj9/test/support/Support_Exec.java

```diff
@@ -153,7 +153,7 @@ public static void checkStderr(Object[] execArgs) {
 			execArgs = new String[baseArgsArray.length + args.length];
 			System.arraycopy(baseArgsArray, 0, execArgs, 0,
 					baseArgsArray.length);
-		} else if (vendor.indexOf("IBM") != -1) {
+		} else if (vendor.indexOf("IBM") != -1 || vendor.indexOf("OpenJ9") != -1) {
 			String full = System.getProperty("java.fullversion");
 			logger.debug("***" + full + "***");
```

Comment:
would also change logger.debug to logger.info - as that information is useful to print to console

Reply:
ok will add that